### PR TITLE
Migrate react and vue to webpack 5

### PR DESCRIPTION
**Pull request by mshima**
Fixes https://github.com/jhipster/generator-jhipster/issues/12712

---

Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) was updated if necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (bellow reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Comment by mraible** [comment]
You're awesome, @mshima! 😃

File: generators/client/templates/react/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "bootswatch": "4.3.1",
     "loaders.css": "0.1.2",
     "lodash": "4.17.20",
+    "path-browserify": "^1.0.1",
```
Comment (DanielFran) [inline generators/client/templates/react/package.json:12]: Used fixed version

**Review by DanielFran — CHANGES_REQUESTED** [review]

File: generators/client/templates/react/package.json (hunk 1 above)
Comment (mshima) [reply generators/client/templates/react/package.json:12]:
```suggestion
    "path-browserify": "1.0.1",
```

**Review by DanielFran — APPROVED** [review]

**Comment by mshima** [comment]
Angular howto added to https://github.com/jhipster/generator-jhipster/issues/13642#issuecomment-763798399.

**Comment by mshima** [comment]
Bounty claimed https://opencollective.com/generator-jhipster/expenses/32632

**Comment by pascalgrimaud** [comment]
@mshima : approved

**Comment by mshima** [comment]
Thanks @pascalgrimaud

**Pull request merged**